### ISSUE TITLE
[BUG] Incorrect accent in Elevenlabs voice output when non-English language is selected

### ISSUE THREAD

**Issue by rwb-truelime** [enricoros/big-AGI#649]
### Description

When selecting a non-English language (specifically Dutch) for voice output using Elevenlabs, the generated speech still has a strong English accent instead of the expected Dutch accent. This issue persists even when a Dutch voice is explicitly selected.

Steps to reproduce:
1. Open big_AGI
2. Navigate to the language selection settings
3. Choose Dutch as the language
4. Select the Dutch voice (voice_id: YUdpWWny7k5yb4QCeweX) for Elevenlabs output
5. Generate voice output for Dutch text

Expected behavior:
The voice output should have a proper Dutch accent matching the selected Dutch voice.

Actual behavior:
The voice output has a strong English accent, despite the Dutch language and voice selection.

Additional information:
- Elevenlabs voice ID used: YUdpWWny7k5yb4QCeweX
- This issue seems to occur consistently when selecting non-English languages

Please investigate why the accent is not changing according to the selected language and voice, and implement a fix to ensure the correct accent is applied for non-English language selections.

### Device and browser

Device: Dell XPS 15 9520
OS: Microsoft Windows 11 Enterprise, Version 10.0.22631 Build 22631
Browser: Microsoft Edge 129.0.2792.52 (Official build) (64-bit)

Additional details:

Processor: 12th Gen Intel(R) Core(TM) i7-12700H, 2300 MHz, 14 core(s), 20 logical processor(s)
RAM: 32.0 GB
System Type: x64-based PC
BIOS Version: Dell Inc. 1.25.0, 11-7-2024
Language: Dutch (based on the system information being in Dutch)

### Willingness to Contribute

- [X] 🙋‍♂️ Yes, I would like to contribute a fix.

**Comment by darthalex2014** [comment]
I'm having the same issue. I even switched to the eleven_turbo_v2_5 model (it supports Russian, unlike eleven_multilingual_v1), but it still speaks with an English accent.

**Comment by enricoros** [comment]
Thanks @rwb-truelime -- does it do the same thing if the language is changed inside Preferences?
![image](https://github.com/user-attachments/assets/6b821a19-8ab9-4ac8-b2e8-99e191102202)

It's possible we are forcing this language in the app. @darthalex2014 does eleven_turbo_v2_5 detect the language automatically? If so then we don't need the UI selector for the language anymore, and we'd rather just call the latest model.

**Comment by rwb-truelime** [comment]
> Thanks @rwb-truelime -- does it do the same thing if the language is changed inside Preferences? ![image](https://private-user-images.githubusercontent.com/32999/371975740-6b821a19-8ab9-4ac8-b2e8-99e191102202.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2Nzk0OTQsIm5iZiI6MTcyNzY3OTE5NCwicGF0aCI6Ii8zMjk5OS8zNzE5NzU3NDAtNmI4MjFhMTktOGFiOS00YWM4LWIyZTgtOTllMTkxMTAyMjAyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTMwVDA2NTMxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwNjQ3YjI4OWNhNmQ5MWJiYTdiNjY2NjAyZGFiMDUwN2Q1M2E2MjZhZTY0YzU3ZTZmYjljYzE5Zjk3ZmI2NzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GUXAwAm9xxoDGojMkVFsjjpZpoXS6ndV_r_xAi9x2A8)
> 
> It's possible we are forcing this language in the app. @darthalex2014 does eleven_turbo_v2_5 detect the language automatically? If so then we don't need the UI selector for the language anymore, and we'd rather just call the latest model.

Yes, that is the place I've set the language to Dutch. Thanks for looking into this!

**Comment by zoollcar** [comment]
I test in Chinese, Use the following code to switch to eleven_turbo_v2_5, I think it can detect the language automatically

src\modules\elevenlabs\elevenlabs.server.ts
```
    const body: ElevenlabsWire.TTSRequest = {
      text: text,
      ...({ model_id: 'eleven_turbo_v2_5' }),
    };
```

It sounds much better than *eleven_multilingual_v1* or *eleven_multilingual_v2*, It still with an English accent but acceptable.

**Comment by enricoros** [comment]
Fixed - ElevenLabs now uses eleven_multilingual_v2 by default which properly handles non-English accents. Users can also explicitly select the model in voice settings.
